README.md

```diff
@@ -1,6 +1,7 @@
 # FastUI
 
 Find the documentation [here](https://docs.pydantic.dev/fastui/).
+Join the discussion in the #fastui slack channel [here](https://pydanticlogfire.slack.com)
```

Comment:
Isn't it better to put the fastui channel link?
```suggestion
Join the discussion in the #fastui slack channel [here](https://pydanticlogfire.slack.com/archives/C0720M7D31S)
```

Reply:
Yes @hramezani. Great idea.